geometry_msgs/Pose2D    pose
int32                   apriltag_id
Corner                  corner
# Parent and Child initalized internally after service call